src/app/(proper_react)/DashboardTopBanner.module.scss

```diff
@@ -0,0 +1,41 @@
+@import "../tokens";
+
+.container {
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  border: 1px solid rgba($color-purple-50, 0.2);
+  border-radius: $border-radius-md;
+  padding: $layout-lg $spacing-xl;
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-sm;
+
+    h3 {
+      font: $text-title-2xs;
+    }
+
+    .cta {
+      margin-top: $spacing-sm;
```

Comment:
Nit, but I'd prefer a `gap` on the parent if possible over a margin on the child, in the spirit of [composable CSS](https://non-traditional.dev/encapsulated-css-the-key-to-composable-layouts-94f11c177cc1): if we consistently keep parents responsible for layout their children, rather than children repositioning themselves sometimes, that makes it easier to debug issues, to move things around, and to understand the flow of our styles.